bin/probot.js

```diff
@@ -15,10 +15,30 @@ program
   .command('run', 'run the bot')
   .command('receive', 'Receive a single event and payload')
 
-program.on('command:*', (cmd) => {
-  console.log(`\nInvalid command ${cmd}\n`)
-  program.outputHelp()  
-})
+// Parsing the exact argument
+const [, , arg] = process.argv
+
+// Valid commands and options in scope
+const availableCommands = program.commands.map(cmd => cmd._name)
+const availableOptions = ['-V', '--version', '-h', '--help']
+
+// Helper method to show suitable warning message
+const showWarningMessage = (type) => {
+  console.log(`Invalid ${type} ${arg}\n`)
+  program.help()  	
```

Comment:
Commander's `.help()` method exits with code 0 - which generally means a program succeeded. In this context, I think we should indicate that Probot errored by exiting with code 1.

```diff
- program.help()
+ program.outputHelp()
+ process.exit(1)
```